pkg/analyze/files/pods/default-unhealthy.json

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
Could we add it inside of a directory called testdata?
Why Golang has a particular meaning for those and for we add mock data that is the best approach.